yb-voyager/cmd/exportDataStatusCommand.go

```diff
@@ -0,0 +1,123 @@
+package cmd
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"path/filepath"
+	"sort"
+	"strings"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"github.com/yugabyte/yb-voyager/yb-voyager/src/utils"
+)
+
+var exportDataStatusCmd = &cobra.Command{
+	Use:   "status",
+	Short: "Print status of an ongoing/completed export.",
+
+	Run: func(cmd *cobra.Command, args []string) {
+		validateExportDirFlag()
+		err := runExportDataStatusCmd()
+		if err != nil {
+			log.Errorf("Get export data status failed: %s", err)
+			fmt.Fprintf(os.Stderr, "error: %s\n", err)
+			os.Exit(1)
+		}
+	},
+}
+
+func init() {
+	exportDataCmd.AddCommand(exportDataStatusCmd)
+}
+
+type exportTableMigStatusOutputRow struct {
+	tableName string
+	status    string
+}
+
+// Note that the `export data status` is running in a separate process. It won't have access to the in-memory state
+// held in the main `export data` process.
+func runExportDataStatusCmd() error {
+	exportSchemaDoneFlagFilePath := filepath.Join(exportDir, "metainfo/flags/exportSchemaDone")
+	_, err := os.Stat(exportSchemaDoneFlagFilePath)
+	if err != nil {
+		if errors.Is(err, os.ErrNotExist) {
+			return fmt.Errorf("cannot run `export data status` before schema export is done")
+		}
+		return fmt.Errorf("check if schema export is done: %w", err)
+	}
+
+	tableMap := make(map[string]string)
+	dataDir := filepath.Join(exportDir, "data")
+	dbTypeFlag := filepath.Join(exportDir, "metainfo", "schema", "source-db-")
```

Comment:
instead of check for each flag. You can re-use this function ExtractMetaInfo():
```
sourceDBType = ExtractMetaInfo(exportDir).SourceDBType
```

After that you can just use switch case statements based on db types.